server/bleep/src/webserver/answer.rs

```diff
@@ -432,7 +438,7 @@ impl Conversation {
             .context("couldn't find conversation title")?;
 
         let exchanges = serde_json::to_string(&self.exchanges)?;
-        let llm_history = serde_json::to_string(&self.llm_history)?;
+        let llm_history = serde_json::to_string(&self.history)?;
```

Comment:
why are we renaming `llm_history` to `history`? and could we remove the `llm_history` nomenclature altogether in that case (this `let` binding for example)?

Reply:
To clean things up, there's no type of history other than an `llm_history`. In future we'll merge this the notion of an `Exchange` so that the history is just a `Vec<Exchange>`.

I'll remove the traces of `llm_history` nomenclature.